### ISSUE TITLE
Creating secrets for an organization 

### ISSUE THREAD

**Issue by tomaswarynyca** [metroline/metroline#22]
In my case I need to share several secrets between all the repositories of the organization is half tedious to go through each repository and add them manually.

**Comment by gempain** [comment]
This is [already](https://docs.metroline.io/ci-configuration/secrets#global-secrets) possible and documented. You can set the `METROLINE_GLOBAL_SECRETS` environment variable.

**Comment by tomaswarynyca** [comment]
Hello, I am not clear on the issue of secrets for an organization. I have several organizations in gitea that can use metroline to deploy, if I put METROLINE_GLOBAL_SECRETS all will be able to access those secrets? I just want to assign secrets with the same name but different ones to different organizations.

**Comment by gempain** [comment]
Oh, I see. Yes, this isn't supported at the moment, and think it would be useful. We would need to add a new collection for this, and a new page in the UI similar to that currently used for secrets, minus the branch configuration.

**Comment by tomaswarynyca** [comment]
![image](https://user-images.githubusercontent.com/41587659/110686352-191c8180-81be-11eb-94b8-c55fd5fb35b2.png) 

Yes, we have to see how to handle it. It can be a menu for organizations and there the administrators can put the global secrets.

**Comment by gempain** [comment]
Yup, good idea, but we'll have to make sure users can't access secrets from orgs they're not a member of. If you look at the `User` model, you'll [see](https://github.com/metroline/metroline/blob/2d0e2b3592db8cdebc583de244dc3cce7f3349f8/src/server/users/user.ts) that we currently don't store this info. We'd need to store it. We have it in memory but lose it [here](https://github.com/metroline/metroline/blob/2d0e2b3592db8cdebc583de244dc3cce7f3349f8/src/server/auth/create-or-update-user.ts#L23).

We could add a page with "Secrets" (in the menu you showed) with one "card" per organization, and in each card a table with secrets (just like the table used for repo secrets, except that you wouldn't be able to configure branch).

**Comment by tomaswarynyca** [comment]
It would be necessary to store a list of the organizations and the role it has, so we can validate if it has to see the tab.
Usually an organization follows a branch standard, so it would be good to implement global and branch secrets.

**Comment by gempain** [comment]
Makes sense, but branches are specific to repos, not branches. Roles would be a good thing to store as well. We already store roles per repo (push / pull / admin). I'm assuming the same info is available for orgs.
